core/common/src/main/java/alluxio/AbstractClient.java

```diff
@@ -228,6 +229,10 @@ private void doHandshake() throws AlluxioStatusException {
         // TODO(binfan): support propagating unsetting properties from master
         if (PropertyKey.isValid(name) && property.getValue() != null) {
           PropertyKey key = PropertyKey.fromString(name);
+          if (!key.getScope().contains(Scope.CLIENT)) {
```

Comment:
Do you want to put this check in the `if`?

Reply:
I need access to the `PropertyKey`